marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -76,6 +76,14 @@ describe('json scenario for snippet on browse documents page', () => {
   });
 
   it('apply facet search and verify docs, hub/entity properties', () => {
+    //verify collapsable slider functionality
+    browsePage.getSideBarCollapseIcon().should('be.visible');
+    browsePage.getSideBarContainer().should('be.visible');
+    browsePage.getSideBarCollapseIcon().click();
+    browsePage.getSideBarContainer().should('not.be.visible');
+    browsePage.getSideBarCollapseIcon().click();
+    browsePage.getSideBarContainer().should('be.visible');
+
```

Comment:
I have pushed an RTL test to verify collapsible sidebar, so you can get rid of this test in e2e.
Please **fetch my changes first**, so your repo gets updated.
git fetch timur-isangulov
git rebase timur-isangulov/DHFPROD-4589

Reply:
Removed. Thanks.